packages/safety-suite-api/src/api/claims/claims.ts

```diff
@@ -256,21 +256,19 @@ export function updateClaim(
  * Delete claim by id.
  *
  * @param claimId - Claim UUID
- * @param groupId - Group ID
  * @param customerAlias - Customer alias.
  * @param apiOptions - Optional options for runApi.
  * @returns - Claim model response object.
  */
 export function deleteClaim(
   claimId: string,
-  groupId: Id,
   customerAlias: string,
   apiOptions?: ApiOptions
 ): Request<EmptyResponse> {
   return runApi({
     method: 'DELETE',
     urlRoot: 'claimsSinkUrlRoot',
-    route: `/api/claims/${claimId}/${groupId}`,
+    route: `/api/claims/${claimId}`,
```

Comment:
I believe we will need to remove the `groupId` param passed here as well for this change -> https://github.com/idelic-inc/safety-suite-kernel/blob/79753939f8f035cefa459b3ba872efa5af5a77cc/src/microfrontends/claims/pages/Claim.tsx#L650

Reply:
Nado approved the PR for it already: https://github.com/idelic-inc/safety-suite-kernel/pull/924 But can you double check that's the same as the code you are mentioning?